src/test/java/gov/cancer/tests/cts/advanced_search_form/CancerTypeSection_Test.java

```diff
@@ -0,0 +1,606 @@
+package gov.cancer.tests.cts.advanced_search_form;
+
+import gov.cancer.framework.ExcelDataReader;
+import gov.cancer.framework.QueryParametersComparator;
+import gov.cancer.pageobject.cts.AdvancedSearchPage;
+import gov.cancer.pageobject.cts.SearchNavigationResult;
+import gov.cancer.tests.TestObjectBase;
+import gov.cancer.tests.TestRunner;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+
+/**
+ * This test class contains all tests related to the Cancer Type/Condition section on the Advanced Search Form
+ *
+ *
+ */
+public class CancerTypeSection_Test extends TestObjectBase {
+  /** Verify that CancerType Section is visible
+   * parameters
+   * @param path       url
+   *
+   */
+  @Test(dataProvider = "getCancerType")
+  public void verifyCancerTypeSectionIsVisible(String path) {
+    TestRunner.run(AdvancedSearchPage.class, path, (AdvancedSearchPage page) -> {
+      //asserting if the Cancer Type Section is displayed
+      Assert.assertTrue(page.getCancerTypeSection().isVisible(), "Age Section is not visible");
+    });
+  }
+
+
+  /** Verify that All button is visible in CancerType Section
+   * parameters
+   * @param path       url
+   *
+   */
+  @Test(dataProvider = "getCancerType")
+  public void verifyAllButtonIsVisible(String path) {
+    TestRunner.run(AdvancedSearchPage.class, path, (AdvancedSearchPage page) -> {
+      //asserting that All button is displayed in the Cancer Type Section
+      Assert.assertTrue(page.getCancerTypeSection().getAllButton().isVisible(), "Age Section is not visible");
```

Comment:
Assert error message should be "All button is not visible"

Reply:
fixed this